ironfish/src/network/messages/pooledTransactions.test.ts

```diff
@@ -20,14 +20,35 @@ describe('PooledTransactionsRequest', () => {
 })
 
 describe('PooledTransactionsResponse', () => {
-  it('serializes the object into a buffer and deserializes to the original object', () => {
+  const nodeTest = createNodeTest()
+
+  function expectPooledTransactionsResponseToMatch(
+    a: PooledTransactionsResponse,
+    b: PooledTransactionsResponse,
+  ): void {
+    // Test transactions separately because Transaction not a primitive type
+    expect(a.transactions.length).toEqual(b.transactions.length)
+    a.transactions.forEach((transactionA, transactionIndexA) => {
+      const transactionB = b.transactions[transactionIndexA]
+
+      expect(transactionA.hash().equals(transactionB.hash())).toBe(true)
+    })
+
+    expect({ ...a, transactions: undefined }).toMatchObject({ ...b, transactions: undefined })
+  }
+
+  // eslint-disable-next-line jest/expect-expect
+  it('serializes the object into a buffer and deserializes to the original object', async () => {
+    const { transaction } = await useTxSpendsFixture(nodeTest.node)
+
     const rpcId = 53242
-    const transactions = [...Array(100)].map((_) => randomBytes(randomInt(500, 10000)))
+    const transactions = [...Array(100)].map((_) => transaction)
```

Comment:
It'd be nice to have 2 unique transactions here.

Reply:
Changed to having 2 unique transactions